dist/images/start-ic-db.sh

```diff
@@ -4,8 +4,51 @@ set -eo pipefail
 TS_NAME=${TS_NAME:-ts}
 TS_CIDR=${TS_CIDR:-169.254.100.0/24}
 
-/usr/share/ovn/scripts/ovn-ctl --db-ic-nb-create-insecure-remote=yes --db-ic-sb-create-insecure-remote=yes start_ic_ovsdb
-/usr/share/ovn/scripts/ovn-ctl status_ic_ovsdb
-ovn-ic-nbctl --may-exist ts-add "$TS_NAME"
-ovn-ic-nbctl set Transit_Switch ts external_ids:subnet="$TS_CIDR"
-tail -f /var/log/ovn/ovsdb-server-ic-nb.log
+rm -f /etc/ovn/ovn_ic_nb_db.db /etc/ovn/ovn_ic_sb_db.db /var/log/ovn/ovsdb-server-ic-nb.log /var/log/ovn/ovsdb-server-ic-sb.log
```

Comment:
container restart will delete the db and lost all data

Reply:
deleting db is deleted now.